### PR TITLE
:tada: php8.0

### PR DESCRIPTION
**Pull request by jerfeson**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM php:8.0-cli
+RUN apt-get update && apt-get install -y apt-utils zip unzip
+# Get latest Composer
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+COPY . /usr/src/slim-twig-view
+WORKDIR /usr/src/slim-twig-view
+CMD [ "php" ]
```
Comment (l0gicgate) [inline Dockerfile]: Can you please remove this file?

**Review by l0gicgate — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (jerfeson) [reply Dockerfile]: of course : )

**Comment by proton-ab** [comment]
Bumping as this is blocking for us

**Review by l0gicgate — CHANGES_REQUESTED** [review]
We need to remove the allowed failures on the nightly builds in the `.travis.yml`

File: composer.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 | ^8.0",
```
Comment (seanhamlin) [inline composer.json]: Pretty sure the syntax for a logical `OR` is a double pipe. See https://getcomposer.org/doc/articles/versions.md#version-range

> You can define multiple ranges. Ranges separated by a space ( ) or comma (,) will be treated as a logical AND. A double pipe (||) will be treated as a logical OR. AND has higher precedence than OR.

```suggestion
        "php": "^7.2 || ^8.0",
```

**Review by seanhamlin — CHANGES_REQUESTED** [review]

**Comment by seanhamlin** [comment]
Also, by the looks TravisCI supports PHP `8.0.0` as well now https://github.com/php-build/php-build/pull/651, so it looks like you can leave `nightly` as is (with the allowed failures), and add in another PHP version to test of `8.0` (no allowed failures).

**Comment by l0gicgate** [comment]
@seanhamlin PHP 8.0 is still not working on Travis as seen here:
https://travis-ci.com/github/slimphp/Twig-View/jobs/455985200

**Pull request closed without merging**